### ISSUE TITLE
Change in Minio credentials does not propagate to 'mlpipeline-minio-artifact' in user namespace

### ISSUE THREAD

**Issue by Barteus** [canonical/minio-operator#32]
I'm not sure which component should be addressed here. I'm filling it against minio because the change starts from here.

When changing the minio creds it is changed in `kubeflow` namespace in secret mlpipeline-minio-artifact but it is not changed in the user's namespace.
A single user installation is also impacted.

**Result**
It is not possible to run workflows using KFP. 
```
This step is in Error state with this message: Error (exit code 1): failed to put file: The Access Key Id you provided does not exist in our records.
```

**Reproduce**:
Deploy Kubeflow and after all is working change the access-key / secret-key for minio.

**Workaround**:
Copy the mlpipeline-minio-artifact from kubeflow namespace to user namespace

**Comment by Barteus** [comment]
Additionally, the secrets are also placed in the annotations as metacontroller.k8s.io/last-applied-configuration

**Comment by Barteus** [comment]
This issue can be solved by https://github.com/canonical/kfp-operators/issues/53

**Comment by ca-scribner** [comment]
Closed by canonical/kfp-operators#53
